dnfdaemon-server/package.hpp

```diff
@@ -45,6 +45,19 @@ enum class PackageAttribute {
     license,
     description,
 
+    // ReldepList attributes
+    provides,
+    requires_all,
```

Comment:
why is it called `requires_all` and sometimes just `requires`?

Reply:
It's because `requires` is a reserved keyword since c++ 20